consensus/core/src/commit_syncer.rs

```diff
@@ -373,6 +373,15 @@ impl<C: NetworkClient> CommitSyncer<C> {
         inner: Arc<Inner<C>>,
         commit_range: CommitRange,
     ) -> (CommitIndex, Vec<TrustedCommit>, Vec<VerifiedBlock>) {
+        // Individual request base timeout.
+        const TIMEOUT: Duration = Duration::from_secs(10);
+        // Max per-request timeout will be base timeout times a multiplier.
+        // At the extreme, this means there will be 120s timeout to fetch max_blocks_per_fetch blocks.
+        const MAX_TIMEOUT_MULTIPLIER: u32 = 12;
+        // timeout * max number of targets should be reasonably small, so the
```

Comment:
should commit_sync_parallel_fetches be considered here as well?

Reply:
`commit_sync_parallel_fetches` apply to the higher level. It is essentially the number of `fetch_loop()` running in parallel.